brew/src/main/java/nl/tudelft/serg/evosql/brew/generator/junit/JUnitGeneratorHelper.java

```diff
@@ -0,0 +1,176 @@
+package nl.tudelft.serg.evosql.brew.generator.junit;
+
+import com.squareup.javapoet.ClassName;
+import com.squareup.javapoet.MethodSpec;
+import com.squareup.javapoet.ParameterizedTypeName;
+import com.squareup.javapoet.TypeName;
+
+import javax.lang.model.element.Modifier;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Builds helper methods for the JUnit generator.
+ */
+public class JUnitGeneratorHelper {
+    /**
+     * The name of the map maker function.
+     */
+    public static final String RUN_SQL_NAME = "runSql";
+
+    /**
+     * The return type of the runSql method.
+     */
+    public static final ParameterizedTypeName RUN_SQL_RETURN_TYPE;
+
+    /**
+     * The name of the map maker function.
+     */
+    public static final String MAP_MAKER_NAME = "makeMap";
+
+    /**
+     * The name of the result column getter function.
+     */
+    public static final String GET_RESULT_COLUMNS_NAME = "getResultColumns";
+
+    /**
+     * The name of the JDBC URL constant.
+     */
+    static final String NAME_DB_JDBC_URL = "DB_JDBC_URL";
```

Comment:
This is very nitpicky however you should maybe change the name of this constant to refer to it more as a name of a constant and not a value passed into the generator. Someone reading this could think that the `DB_JDBC_URL` parameter of the DB will now always be generated as `"DB_JDBC_URL"`.

Reply:
I agree, and I guess that applies to the other constants too then. I'm not sure how to name them though.

Reply:
Would adding `_CONSTANT` as a suffix be sufficient?

Reply:
I'd rather have it as `FIELD_DB_JDBC_URL` for the fields and `METHOD_RUN_SQL` then, would that be OK?

Reply:
Sure!